state-synchronizer/src/coordinator.rs

```diff
@@ -211,74 +205,16 @@ impl<T: ExecutorProxyTrait> SyncCoordinator<T> {
             )
             .await
             .expect("[state sync] failed to execute empty blocks");
-            self.sync_request = Some(request);
-            self.check_client_request().await;
-        } else {
-            let peers = request.target.signatures().keys().copied().collect();
-            self.peer_manager.set_peers(peers);
-            self.sync_request = Some(request);
-            self.request_next_chunk(0).await;
-        }
-    }
-
-    async fn check_client_request(&mut self) {
```

Comment:
what is this used for?

Reply:
This was used for checking that the client state sync request is ready. "Readiness" in this case was determined by the fact that the storage has been synced to a value that is higher or equal the request target.
In the old flow we used to have this check in the `commit()` function, which is what we're doing here as well.